airflow/serialization/serialized_objects.py

```diff
@@ -27,6 +27,8 @@
 import pendulum
 from dateutil import relativedelta
 
+from airflow.utils.timezone import _get_tzname_or_offset
```

Comment:
```suggestion
from airflow.utils.timezone import get_tzname_or_offset
```

If it's called from outside the module it shouldn't have `_` prefix.